web/src/blueslip_stacktrace.ts

```diff
@@ -123,7 +123,7 @@ export async function display_stacktrace(ex: unknown, message?: string): Promise
                     "constructor" in prototype
                         ? `thrown ${prototype.constructor.name}`
                         : "thrown",
-                message: ex === undefined ? message : String(ex),
+                message: ex === undefined ? message : `${String(ex)}: ${message}`,
```

Comment:
This will be confusing if we’re not in the first iteration of the loop due to `ex.cause` chaining.